scripts/import_srpm.py

```
@@ -76,7 +81,7 @@ def main():
     print(" extracting SRPM...")
 
     os.chdir('SOURCES')
-    os.system('rpm2cpio "%s" | cpio -idmv' % source_rpm_abs)
+    call_process(['sh', '-c', 'rpm2cpio "%s" | cpio -idmv' % source_rpm_abs])
```

```suggestion
    call_process(['sh', '-o', pipefail', '-c', 'rpm2cpio "%s" | cpio -idmv' % source_rpm_abs])
```

otherwise it won't fail if rpm2cpio fails.
Or we can use #743 — I didn't see there was already a PR for that problem 😅 
It's in pure Python, but it's a much bigger change.

---

I'm sorry, `pipefail` is a bash option, it doesn't exist on posix shell. I've actually look before writing my commit message, saying that this ignores error from `rpm2cpio`.
We don't know what implementation is used for `sh`, it might be `dash` on Debian, or `bash` on Arch Linux.